src/main/java/seedu/address/storage/XmlAdaptedGroup.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import javax.xml.bind.annotation.XmlElement;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.UniqueList;
+import seedu.address.model.group.Group;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * JAXB-friendly version of the Group.
+ */
+public class XmlAdaptedGroup {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Group's %s field is missing!";
+
+    @XmlElement(required = true)
+    private String name;
+    @XmlElement(required = true)
```

Comment:
so description is required when people make a group now?

Reply:
Yes it is for now.

Reply:
If description is not entered, the default description is "No group description has been inputted"